master/internal/db/postgres_snapshots_test.go

```diff
@@ -56,5 +59,6 @@ func TestCustomSearcherSnapshot(t *testing.T) {
 	queue2, err := cSearcher2.GetCustomSearcherEventQueue()
 	require.NoError(t, err)
 	require.Equal(t, queue1.GetEvents(), queue2.GetEvents())
-	db.DeleteSnapshotsForExperiment(1)
+	db.DeleteSnapshotsForExperiment(exp.ID)
+	db.DeleteExperiment(exp.ID)
```

Comment:
Small improvement in postgres_snapshots_test: Since, I'm deleting snapshot it's also better to delete experiment otherwise locally while rerunning make devcluster you can get into issues.